Sources/BrowserServicesKit/PrivacyConfig/AppPrivacyConfiguration.swift

```diff
@@ -88,14 +96,71 @@ public struct AppPrivacyConfiguration: PrivacyConfiguration {
         default: return false
         }
     }
+    
+    private func rolloutEnabled(subfeature: any PrivacySubfeature,
+                                rollouts: [PrivacyConfigurationData.PrivacyFeature.Feature.Rollout],
+                                randomizer: (Range<Double>) -> Double) -> Bool {
+        let defsPrefix = "config.\(subfeature.parent.rawValue).\(subfeature.rawValue)"
+        if userDefaults.bool(forKey: "\(defsPrefix).\(Constants.enabledKey)") {
+            return true
+        }
+        
+        var willEnable = false
+        if let rolloutSize = userDefaults.value(forKey: "\(defsPrefix).\(Constants.lastRolloutCountKey)") as? Int {
+            guard rolloutSize != rollouts.count else { return false }
```

Comment:
I wonder here whether we should check that the new rollout size is larger than the old one, rather than just different.

The situation I have in mind (and it's an edge case for sure) is that we deploy a new rollout value - but quickly remove it. With the current implementation, that means that any clients that got a new config in the window where the new rollout value was available will update again and have yet another chance to roll.

By making this `newRolloutSize > oldRolloutSize`, then we can at least quickly remove a rollout value to prevent new clients from getting the new rollout value, but any clients that managed to get it won't re-roll.